solidity/contracts/client/GasRouter.sol

```diff
@@ -59,6 +59,13 @@ abstract contract GasRouter is Router {
     function quoteGasPayment(
         uint32 _destinationDomain
     ) external view virtual returns (uint256) {
+        return _quoteGasPayment(_destinationDomain);
+    }
+
+    /// @dev Lets derived contracts access the quote
+    function _quoteGasPayment(
+        uint32 _destinationDomain
+    ) internal view virtual returns (uint256) {
```

Comment:
alternatively you could make the `quoteGasPayment` public

Reply:
Is there a rule of thumb for this? because I thought that option as well, but it was hard to choose :sweat_smile: